ignite/services/chain/chain.go

```diff
@@ -359,20 +357,17 @@ func (c *Chain) KeyringBackend() (chaincmd.KeyringBackend, error) {
 		return c.options.keyringBackend, nil
 	}
 
-	config, err := c.Config()
+	// 2nd.
```

Comment:
What do these comments mean?  

Can we change them to something clearer?

Reply:
I understand that these comments where added to indicate each of the possible keyring backend name locations. Personally I think we could remove them. WDYT @aljo242? Should I extend the comments or just remove them?

Reply:
I think we should extend the comments

Reply:
Changeset 6f34c37